### PR TITLE
Added Sign in button on top right of landing page that links to sign in page for supabase

### PR DESCRIPTION
**Pull request by Pythonwithsean**

### CODE REVIEW

**Comment by Pythonwithsean** [comment]
> Vercel does not allow me to Deploy for some reason and i approved

File: components/ui/header.tsx [hunk 1]
```diff
@@ -40,9 +38,18 @@ export default function Header() {
...
+                  className="font-medium text-purple-600 hover:text-gray-200 px-4 py-3 flex items-center transition duration-150 ease-in-out"
+                >
+                  Signin
+                </Link>
+              </li>
             </ul>
           </nav>
-          {process.env.NODE_ENV !== 'production' && <AuthButton />} {/* AuthButton is hidden in production */} 
+          {process.env.NODE_ENV !== "production" && <AuthButton />}
+          {/* AuthButton is hidden in production */}
```
Comment (Ellahinator) [inline components/ui/header.tsx:52]: We already have the auth buttons implemented. This change is not needed.

Comment (Pythonwithsean) [reply components/ui/header.tsx:52]: No problem

Comment (Pythonwithsean) [reply components/ui/header.tsx:52]: Please update Issues, so other contributors don't think It's still an issue @Ellahinator

File: components/ui/header.tsx [hunk 2]
```diff
@@ -40,9 +38,18 @@ export default function Header() {
                   About
                 </Link>
               </li>
+              <li>
+                <Link
+                  href="/signin"
+                  className="font-medium text-purple-600 hover:text-gray-200 px-4 py-3 flex items-center transition duration-150 ease-in-out"
+                >
+                  Signin
```
Comment (Fryingpannn) [inline components/ui/header.tsx:46]: we already have sign in button, can remove this

**Pull request closed without merging**